chrome/manifest.json

```diff
@@ -64,6 +64,7 @@
 		"*://backend.deviantart.com/oembed?url=*",
 		"*://api.gyazo.com/api/oembed*",
 		"*://codepen.io/api/oembed*",
-		"*://api.tumblr.com/v2/blog/*/posts?api_key=*&id=*"
+		"*://api.tumblr.com/v2/blog/*/posts?api_key=*&id=*",
+		"https://xkcd.com/*"
```

Comment:
Couldn't this be made a bit more restrictive, like the other permissions above it? (I.E. `https://xkcd.com/*/info.0.json`)